src/guides/v2.3/release-notes/release-notes-2-3-5-open-source.md

```diff
@@ -518,56 +528,57 @@ We have fixed hundreds of issues in the Magento 2.3.5 core code.
 
 <!--- MC-23228-->
 
-*  The graphical orders chart accessible from the Orders tab on the Admin dashboard now accurately reflects order quantity.
+*  The graphical orders chart accessible from the Orders tab on the Admin now accurately reflects order quantity.
 
 <!--- MC-30461-->
 
 *  Product price change alert email now includes the correct product price. Previously, this email suggested a new product price of 0.
 
 <!--- MC-29111-->
 
-*  You can now save and duplicate all CMS pages. Previously Magento threw this exception when you tried to duplicate certain pages: `Unique constraint violation found`.
+*  You can now save and duplicate all CMS pages. Previously, Magento threw this exception when you tried to duplicate certain pages: `Unique constraint violation found`.
 
 <!--- MC-29994-->
 
-<!--- MC-30261-->
-
-*  You can now add a child product of a grouped product to your cart when one of the grouped product’s other child products is out-of-stock. Previously, when one child product was out-of-stock, you could not add any other child products to the cart.
+*  Magento now redirects you to the home page of the appropriate store view when you change language on CMS pages in a multistore deployment. Previously, Magento displayed a 404 page when you changed language on certain CMS pages.
 
 ### Import/export
 
-<!--- MC-21727-->
-
 <!--- MC-31470-->
 
+*  Magento now successfully imports customer data  using the **Customer and Addresses (single file))** option when `cron` is enabled and the Customer Grid Indexer is set to **Update By Schedule**.  After `cron` executes, the imported customer information is available in the Admin as expected.  Previously, Magento imported the customer data, but did not update the customer grid with the newly imported customer records.
+
 <!--- MC-29361-->
 
 *  Magento now updates images as expected when you use the `hide_from_product_page` setting when importing products in deployments with multiple store views.
 
 <!--- MC-30066-->
 
-*  Magento now deletes temporary files from `/var` as expected after product import has completed.
+*  Magento now deletes temporary files from `<Magento_home>/var` as expected after product import has completed.
 
 <!--- MC-21727-->
 
-*  Magento now removes related, up-sell, and cross-sell products as expected in the `import.csv` file when you set the value of the **Empty attribute value constant** field to `_EMPTYVALUE_` for products in **System** > **Import**. Previously, cross-sell, up-sell, and related products were not removed from the `import.csv` file.
+*  Magento now removes related, up-sell, and cross-sell products as expected in the `import.csv` file when you set the value of the **Empty attribute value constant** field to `_EMPTYVALUE_` for products in **System** > **Import**. Previously, cross-sell, up-sell, and related products were not removed from the import `.csv` file.
```

Comment:
The first `import.csv` should probably also be changed to "import `.csv` file".

Reply:
corrected in source -- thanks